doc/vxlan/EVPN/EVPN_VxLAN_Multihoming.md

```diff
@@ -0,0 +1,1671 @@
+### 3.2.1 CONFIG DB
+
+**EVPN_ETHERNET_SEGMENT**
+
+```
+;New table
+;Specifies EVPN Ethernet Segment interface
+;
+; Status: stable
+key = EVPN_ETHERNET_SEGMENT|"ifname"if_id
+; field = value
+esi        = "AUTO"  or es_id
+                ; es_id is 10 byte colon (:) separated string when type = "TYPE_0_OPERATOR_CONFIGURED".
+                ; Otherwise, esi value should be "AUTO" falling back on ESI type 1 or 3.
+type       = esi_type
+                ; esi_type should be string with one of the below values:
+                ; "TYPE_0_OPERATOR_CONFIGURED" for Type-0 ESI
+                ; "TYPE_1_LACP_BASED" for Type-1 ESI
+                ; "TYPE_3_MAC_BASED" for Type-3 ESI
+ifname     = "ifname"if_id
+                ; if_id is the interface identifier. Same value as in the key.
+                ; e.g., PortChannel1, where the ifname is PortChannel and the if_id is 1.
+df_pref    = 1*5DIGIT
+                ; Designated-Forwarder election preference for this router in (1..65535) range.
+                ; Default=32767.
+```
+
+**PORTCHANNEL**
+```
+; Existing table
+; Add system-id-mac field for PortChannel interfaces
+;
+; Status: stable
+key = PORTCHANNEL|"PortChannel"po_id
+; field = value
+system_mac = mac_address
+				; New field. 6-byte colon(:) separated LACP system mac address
+```
+
+**EVPN_MH_GLOBAL**
+
+```
+; New table
+; Specifies EVPN Multihoming global parameters
+;
+; Status: stable
+key = EVPN_MH_GLOBAL|default
+; field = value
+startup_delay  = 1*4DIGIT
+				; Amount of time in secs (0..3600 range) to wait
+				; before bringing up Multihomed ES. Default=300.
+                                ; Value 0 means disabled.
+mac_holdtime   = 1*5DIGIT
+				; Amount of time in secs (0-86400 range) to wait
+				; before aging out peer VTEPs MAC entry after it
+				; has been withdrawn. Default=1080.
+                                ; Value 0 means disabled.
+neigh_holdtime = 1*5DIGIT
+				; Amount of time in secs (0-86400 range) to wait
+				; before aging out peer VTEPs ARP/ND entry after
+				; it has been withdrawn. Default=1080.
+                                ; Value 0 means disabled.
+```
+
+<a id="322-APP-DB"></a>
+### 3.2.2 APP DB
+
+**EVPN_SPLIT_HORIZON_TABLE**
+
+```
+; New table
+; Specifies split-horizon filtering source VTEPs for EVPN Multihomed interface
+; Producer: fpmsyncd
+; Consumer: shlorch
+; Status: stable
+key               = EVPN_SPLIT_HORIZON_TABLE:"Vlan"vlan_id:"ifname"if_id
+						; vlan_id is 1-4 DIGIT Vlan ID.
+						; if_id is interface number
+; field = value
+vteps            = vtep_list
+						; String of Comma(,) separated list of VTEP IP addresses.
+```
+
+**EVPN_DF_TABLE**
+```
+; New table
+; Specifies designated-forwarder election for this router.
+; Producer: fpmsyncd
+; Consumer: evpnmhorch
+; Status: stable
+key               = EVPN_DF_TABLE:"Vlan"vlan_id:"ifname"pif_id
+						; vlan_id is 1-4 DIGIT VLAN ID.
+						; if_id is interface number
+; field = value
+df        = True
```

Comment:
EVPN_SPLIT_HORIZON_TABLE table is converted to an SAI Isolation Group. EVPN_DF_TABLE adds a new attribute to the Bridgeport attributes (of the L2_ECMP_GROUP type): SAI_BRIDGE_PORT_ATTR_NON_DF. Neither in the first nor in the second case, a binding to VLAN is not necessary.

Reply:
In the case of single-active services, this is necessary.